From c44a9923895697eddffae83e08327b78cf307ee4 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Mon, 29 Apr 2019 17:13:59 -0700
Subject: [PATCH] TRITON-1637 DAPI tests fail in
 hard-filter-volumes-from.test.js Reviewed by: John Levon
 <john.levon@joyent.com> Approved by: John Levon <john.levon@joyent.com>

---
 test/algorithms/hard-filter-volumes-from.test.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/algorithms/hard-filter-volumes-from.test.js b/test/algorithms/hard-filter-volumes-from.test.js
index 21e655c..71d1bf9 100644
--- a/test/algorithms/hard-filter-volumes-from.test.js
+++ b/test/algorithms/hard-filter-volumes-from.test.js
@@ -96,7 +96,7 @@ test('filterVolumesFrom() with no getVm set', function (t) {
 
 
 test('filterVolumesFrom() with getVm set', function (t) {
-	common.OPTS.getVm = function (opts, cb) {
+	common.OPTS.getVm = function (opts, _extra, cb) {
 		return (cb(null, VM_LOOKUP[opts.uuid]));
 	};
 
-- 
2.21.0

